app/controllers/miq_ae_class_controller.rb

```diff
@@ -1660,6 +1661,20 @@ def x_show
     tree_select
   end
 
+  def refresh_git_domain
+    if params[:button] == "save"
+      git_based_domain_import_service.import(params[:git_repo_id], params[:git_branch_or_tag])
```

Comment:
Wouldn't this allow any user to spoof and consume someone else's repository (potentially private)?